newsletter.go

```diff
@@ -118,9 +124,53 @@ const (
 	mutationCreateNewsletter       = "6234210096708695"
 	mutationUnfollowNewsletter     = "6392786840836363"
 	mutationFollowNewsletter       = "9926858900719341"
+
+	// desktop & mobile
+	queryFetchNewsletterDesktop        = "9779843322044422"
```

Comment:
Are the query IDs really different 🤔
The web here are probably outdated

Reply:
It looks like they are btw, but I found no reference to the desktop ones in the web version